tests/field/test_mgrid.py

```diff
@@ -0,0 +1,65 @@
+#!/usr/bin/env python
+
```

Comment:
This is better a as regression/integrated test. For unit tests, each method of the class needs to have a separate test.